From 8b00d2710cd7d01b58c467e1a30a019c3a24db41 Mon Sep 17 00:00:00 2001
From: Anurag Kumar Vulisha <anurag.kumar.vulisha@xilinx.com>
Date: Tue, 24 Feb 2015 20:24:42 +0530
Subject: [PATCH 401/532] mtd:spi-nor: Added support for issi flash parts

commit 9cb4f1239319f78856ef4b948c8fc4bdd6f6d040 from
https://github.com/Xilinx/linux-xlnx.git

 Added support for issi flash parts is25lp128, is25lp064 & is25lp032

Signed-off-by: Anurag Kumar Vulisha <anuragku@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 include/linux/mtd/cfi.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/mtd/cfi.h b/include/linux/mtd/cfi.h
index 9b57a9b..d4a1768 100644
--- a/include/linux/mtd/cfi.h
+++ b/include/linux/mtd/cfi.h
@@ -371,6 +371,7 @@ struct cfi_fixup {
 #define CFI_MFR_HYUNDAI		0x00AD
 #define CFI_MFR_INTEL		0x0089
 #define CFI_MFR_MACRONIX	0x00C2
+#define CFI_MFR_ISSI		0x009d
 #define CFI_MFR_NEC		0x0010
 #define CFI_MFR_PMC		0x009D
 #define CFI_MFR_SAMSUNG		0x00EC
-- 
2.7.4

